modules/core/pom.xml

```diff
@@ -330,7 +330,11 @@
             <groupId>org.apache.axis2</groupId>
             <artifactId>axis2-transport-local</artifactId>
         </dependency>
-        <!--  JSON Schema validator dependency -->
+		<dependency>
```

Comment:
Is there an indentation issue?

Reply:
No indentation issues shown in the local IDE.